ChangeLog

```diff
@@ -1,3 +1,9 @@
+Changes in squid-5.0.3 (04 Jun 2020):
+
+	- Bug 5046: FreeBSD lacks open(2) O_DSYNC flag
+	- ... and several debug improvements
+	- ... and all fixes from 4.12
```

Comment:
IMHO, we should list changes without referring to other Squid releases because the current approach is 

* inconvenient for the reader (who has to follow one or more links to get the complete picture for each release)

* often arguably incorrect or misleading (e.g., one of the `fixes from 4.12` referenced on this line actually belongs to the already released `Changes in squid-5.0.2` set rather than the currently documented `Changes in squid-5.0.3` set; look for "Bug 5030").

I do not insist on this change.

Reply:
This is an indicator for when changes in the stable release sync up with the beta release. The previous few have been done in sync for security fixes, but normally the different release requirements result in the series not matching up for periods of time. That it helps to cut down changelog size and multiple entries for commits is a useful addition.

You can see examples of this in the v4.0.12 to 4.0.14 period (a normal 4 vs 8 week difference), and the 4.0.21 to 4.2 period (the EOL security bump on deprecated v3.5).

Complete lists of every little commit can be found in our changesets page and the repository log.

Reply:
> This is an indicator for when changes in the stable release sync up with the beta release.

It might be a useful indicator for the select few initiated, but it does not offset the negatives enough IMHO.

> Complete lists of every little commit can be found

I know, but this problem is not the "lack of complete lists of every little commit".